### PR TITLE
[Particle Mechanics] Making python functions deprecated

### PR DESCRIPTION
**Pull request by bodhinandach**
As described in the previous PR #2212 that all the functions are no longer used but we want to keep it in the repository in case anybody wants to implement the updated functions in the future.

### CODE REVIEW

**Comment by bodhinandach** [comment]
ping @antonialarese, waiting for the approval.

**Review by antonialarese — APPROVED** [review]

**Pull request merged**
